.github/workflows/tools.yml

```diff
@@ -0,0 +1,44 @@
+name: Build Tools
+
+on:
+  workflow_dispatch:
+    inputs: 
+      tag:
+        description: 'Tagged commit to build tools against'
+        required: true
+        type: string
+
+jobs:
+  docker-push:
+    name: Push to container registry
```

Comment:
I don't think we are pushing containers to registry, maybe we can rename it to `build-boot-tools`?